tests/e2e/testfiles/scenarios.yaml

```diff
@@ -1,69 +1,53 @@
-# List of E2E Tests Scenarios
-
-- scenario: "Install Authorization Proxy Server V1"
+#List of E2E Tests Scenarios
+- scenario: "Install Authorization Proxy Server"
   paths:
-    - "testfiles/authorization-templates/csm_authorization_proxy_server_n_minus_1.yaml"
+    - "testfiles/authorization-templates/csm_authorization_proxy_server.yaml"
```

Comment:
We should continue to use n-1 YAML as testfiles/authorization-templates/csm_authorization_proxy_server.yaml" is used for Stateless Authorization deployment. We need to have separate scenarios for V1 and V2.

Reply:
Done